### DISCUSSION TITLE
How to prevent fraud and Scam

### DISCUSSION THREAD

**Discussion by escamoteur** [DevsHelpDevs/DevsHelpDevs#3] [Q&A]
The moment this platform goes live and gets some popularity we sadly have to expect that people try to use the platform to scam potential donors without being interested in devloping software nor being in any need.

What are your thoughts on how we can minizise scams as much as possible. I'm aware that we won't be able to avoid it to 100% but 90% should be our goal.

What are your ideas how we best counter this danger?

**Comment by escamoteur** [comment]
Here are some of my initial ideas on this topic:

* When an applicant applies for a donation they have to upload a picture of an official ID document, also they have to provide their phone number and email adress which has to be verified
* In every receiver country that wants to participate has to be a group from the dev community who will review applications, getting in touch with the applicant to verify that they are really in need and that they need a working laptop for their job/study.
* Instead of transfering money we should try to use local trustworthy dealers as partner that receive the money and ship the part/computer

**Reply by biggaji** [reply]
The second idea is okay, whereby the applicant can be verified by  a dev community within the area, to find out if it isn't a scam.

**Reply by wilsonowilson** [reply]
I think a combination of the first and the second will do.

**Reply by biggaji** [reply]
Yes it would!

**Reply by StephenLadoye** [reply]
We should try to use local trustworthy dealers and applicants should go through verification process.

**Reply by 805702** [reply]
I'm totally up with idea #2 and #3... they stand solid for me.
The local community will always know better on how to certify that.

but then, how do we bring in communities to the platform and where do we get these ambassadors who will do the screening job for us?

**Reply by escamoteur** [reply]
I would try to reach out to GDGs for a start. Also I imagine that we can recruit applicants that already got a donation. I myself (if I get a nice App 😉 ) will help too to screen applications and hopefully some of you too.

**Reply by higorlapa** [reply]
We could use [Persona](https://pub.dev/packages/persona_flutter) for the proof of life.

**Reply by escamoteur** [reply]
Interesting idea, I'm just not sure if out target audience would get approved by them

**Reply by escamoteur** [reply]
@higorlapa could you do a bit of research how it works?

**Reply by higorlapa** [reply]
The user has to upload the ID (front and back) and also take a selfie. 
they will verify if the user who's sending the ID is the same who's taking the selfie and also extract the data from the ID.  they will verify if it is a valid ID too.
[Maybe they would give us a free plan...](https://withpersona.com/)

**Reply by escamoteur** [reply]
I wonder if this works for Nigeria. If you want to join our Slack, here is the Link:

https://join.slack.com/t/devshelpdevs/shared_invite/zt-nt662pjn-RuaamoGvjSbHhEqtfRXVIg

**Reply by higorlapa** [reply]
Sure. 👍

**Reply by escamoteur** [reply]
https://join.slack.com/t/devshelpdevs/shared_invite/zt-nt662pjn-RuaamoGvjSbHhEqtfRXVIg

**Comment by seunbayo** [comment]
I think asides from the verification of an official id or address verification.

Our target should be those who have already started learning something, not a person with only aspirations as it would help us call for what the individual is already doing.

If there is any tech community that he/she has joined (or people in tech who can stand on his behalf)

We should also have a policy of after some years 2 - 3, The person should also be able to donate at least a certain amount to the platform to support others too.

If the person is supposed to have bought a laptop(probably from side gigs or jobs) he/she can give another recommended member of the community that laptop we gave out.

Let me stop here
